src/Pos/Block/Logic/Slog.hs

```diff
@@ -159,14 +159,15 @@ slogApplyBlocks blunds = do
     -- Note: it's important to put blunds first
     mapM_ dbPutBlund blunds
     -- If the program is interrupted at this point (after putting on block),
-    -- we will rollback all wallet sets at the next launch.
-    onApplyBlocks blunds `catch` logWarn
+    -- we won't save any blocks.
+    callbackBlocks <- onApplyBlocks blunds
```

Comment:
@pva701 I thinks he meant line 161-162 (look at the previous version of the code). But I don't know what to do about the comments, but it's a good idea to see if you need to move this comment elsewhere.